pypesto/optimize/optimizer.py

```diff
@@ -555,3 +560,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 10000}
+        self.options = options
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        sigma0 = 0.25 * np.median(ub)
+        self.options['bounds'] = [lb, ub]
+
+        if cma is None:
+            raise ImportError(
+                "This optimizer requires an installation of cma.")
+
+        result = cma.CMAEvolutionStrategy(
+            x0, sigma0, inopts=self.options
+        ).optimize(problem.objective.get_fval).result
```

Comment:
identation?

Reply:
What indentation would you prefer?

Reply:
Would rather have gone for sth like this, but I would not be strict on this :)
```python
result = cma.CMAEvolutionStrategy(x0, 
                                 sigma0, 
                                 inopts=self.options
                        ).optimize(problem.objective.get_fval).result

```

Reply:
I fixed it